src/templates/masthead.html

```diff
@@ -0,0 +1,12 @@
+<div class="usa-navbar">
+  <button class="usa-menu-btn">Menu</button>
```

Comment:
Just as a secondary note, not relevant to this PR, the `usa-menu-btn` class seems a little out of place. It doesn't follow the usual naming convention for buttons of being `button` rather then shortened `btn` and it's confusing where it's supposed to be as a component ('menu' or 'button').